lib/ace/ext/beautify.js

```diff
@@ -34,15 +34,178 @@ define(function(require, exports, module) {
 "use strict";
 var TokenIterator = require("ace/token_iterator").TokenIterator;
 
-var phpTransform = require("./beautify/php_rules").transform;
```

Comment:
do we need to delete https://github.com/ajaxorg/ace/blob/master/lib/ace/ext/beautify/php_rules.js if it is not used anymore?

Reply:
yes